backend/fms_core/management/commands/_delete_individual.py

```diff
@@ -0,0 +1,86 @@
+from django.apps import apps
+import reversion
+import json
+import logging
+from reversion.models import Version
+from fms_core.models import Individual
+
+# Parameters required for this curation
+ACTION = "action"                     # = delete_individual
+CURATION_INDEX = "curation_index"     # Number indicating the order in which this action was performed during the curation.
+COMMENT = "comment"                   # An optional comment to be stored in the logs
+INDIVIDUAL_NAMES = "individual_names" # An array of individual name that are to be deleted.
+USER_ID = "requester_user_id"         # The user id of the person requesting the curation. Optional. If left empty, uses biobankadmin id.
+
+# Curation params template
+# { CURATION_INDEX: 1,
+#   ACTION: "delete_individual",
+#   COMMENT: "Dr. No asked to delete these individuals that are not used.",
+#   INDIVIDUAL_NAMES: ["Roger", "TiPaul", "Gertrude"], # List of individual names (unique) to be deleted
+#   USER_ID: 5
+# }
+
+# This curation deletes only individual that are not referenced. Remove individual from samples references using "update_field_value" first.
+# To delete parents, a first curation need to be run to delete the child individual (or an update), then parents will be unreferenced.
+
+# function that checks the references to an individual and list them.
+def references(individual):
```

Comment:
Most of my comments will be more stylistic/naming/performance related :P they are not bugs but would  improve the style 

references is too vague for a method name

Reply:
list_references then... ? I do not find it vague personally. Give me a better name if you have one.

Reply:
entities_references or reference_entities or references_to_entities (or something like that) it just doesnt say reference for/from what? 

also for code modularity and responsability it should return what it is (the obj themselves)

Reply:
def list_references_to(individual) ... how does it sound ?

Reply:
better :P